src/FSLibrary/StellarCoreHTTP.fs

```diff
@@ -62,7 +64,9 @@ type LoadGen =
           ("spikesize", self.spikesize.ToString())
           ("spikeinterval", self.spikeinterval.ToString())
           ("batchsize", self.batchsize.ToString())
-          ("offset", self.offset.ToString()) ]
+          ("offset", self.offset.ToString())
+          ("maxfeerate", (if self.maxfeerate.IsSome then self.maxfeerate.Value.ToString() else ""))
+          ("skiplowfeetxs", (if self.skiplowfeetxs then "1" else "0")) ]
```

Comment:
need to update this to be consistent with the new format (true/false instead of 1/0)

Reply:
Thanks for the reminder, fixed

Reply:
Good